accessibility/README.md

```diff
@@ -1,15 +1,101 @@
 # Accessibility
 
-Accessibility **must** be a core consideration in all products that we build.
-
-* [Introduction](introduction.md)
-* [Developing accessible products](developing-accessible-products.md)
-* [Tools](tools.md)
-* [Accessibility checklist](accessibility-checklist.md)
-* [Accessibility in emails](accessibility-in-emails.md)
-* [Accessibility in PDFs](accessibility-in-pdfs.md)
-* [Effective colour contrast](effective-colour-contrast.md)
-* [Common remediations](common-remediations/README.md)
-* [Alt text](alt-text.md)
-
-[Main table of contents](../README.md#table-of-contents)
+* [General principles](#general-principles)
```

Comment:
Dear Hollie, your changes even improve the readability of this PR! I love it. I just wonder whether it is on purpose that the links to all these documents (of which you changed the names really beautifully!) are now gone?

Reply:
> This is due to the fact that I am used to seeing the codebases on GitHub

Yeah, this is basically where my disconnect is - I share these files with people who aren't used to Github, and who don't have the same "sidebar-blindness" (for want of a better term) than you do. 

The ToC links were always a bit of a bodge. The point of them was so we could put the files into a sensible reading order, as the experience was a bit unguided without it. Nobody was brave enough to rename all the docs in the repo to give them numbers, so we added a ToC in each README instead. But now the bodge is _visibly_ a bodge, especially to people who expect to see content on a webpage, rather than a bunch of files. 

I do take your point, though, and I was in two minds last night about whether I should incorporate the links directly into the intro, with a bit of explanation around them. On the one hand it makes the already fairly long introduction even longer... but on the other hand I think the intro could benefit from more signposting. 

I think what I'll do is add an orientation section near the end, fold the "Resources" and "Tools" headings into it, and explicitly direct people to useful pages. That'll mean it does the same job that the list did, but without the visual display being a literal crime against sense and good taste. Will update the PR shortly, thank you! 🙏

Reply:
Ok, I understand. I really appreciate your effort and think that this appendix with "useful links" would improve the value of the README.